cpu/lm4f120/cpu.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2015 Rakendra Thapa <rakendrathapa@gmail.com
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     cpu_lm4f120
+ * @{
+ *
+ * @file
+ * @brief       Implementation of the CPU initialization
+ *
+ * @author      Rakendra Thapa <rakendrathapa@gmail.com>
+ * @}
+ */
+
+#include "cpu.h"
+#include "irq.h"
+#include "kernel.h"
+#include "kernel_internal.h"
+#include "sched.h"
+#include "thread.h"
+#include "arch/thread_arch.h"
+#include "arch/irq_arch.h"
+
+/**
+ * @brief Initialize the CPU, set IRQ priorities
+ */
+void cpu_init(void)
+{
+    // initializes the Cortex-M core
+    cortexm_init();
+
+    /* initialize the clock system */
+    cpu_clock_init(CLK40);
```

Comment:
Don't call this function with a constant -> assign this to a define in the `periph_conf.h` and use that define here: e.g. 
```c
/* periph_conf.h: */
#define CLOCK_SOURCE   CLK40`
```
and s/cpu_clock_init(CLK40)/cpu_clock_init(CLOCK_SOURCE)/